block-node/test-specifications/local-persistence/block-as-local-file-persistence-tp.md

```diff
@@ -0,0 +1,85 @@
+| E2ETC_BLFP_0002 | `Verify Content of Block Persisted Under Unverified Root`                          | `BlockAsLocalFile Persistence` will initially persist a `Block` under `unverified` root. We must make sure that the persisted `Block`'s content is indeed what we expect it to be (same as the content received through the `BlockStream` as input).                                                                                                                                                                                                                           | The persisted `Block`'s content is the same as the content that has been received as input.                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | Send `BlockItem`s for `Block` with `BlockNumber`=12345L to the persistence service.                                                                                                                                                                                                              | Content of persisted `Block` is the same as the one received via input.                                                                                                                                                                                                                                                                                  |         N         |
```

Comment:
Q: Is there a consideration for the order of block items when confirming contents? Or does it make sense to check elsewhere?

Reply:
I see.
Not necessarily block contents proof.
More so if the input to the server is a batch of list of block items in order, then it would make sense to confirm that when grouped and written as a file that the order was maintained and the file is correct.